{{cookiecutter.repo_name}}/src/components/Footer.jsx

```diff
@@ -0,0 +1,29 @@
+import { Paper, Typography } from '@material-ui/core';
+import PropTypes from 'prop-types';
+
+import Link from '@material-ui/core/Link';
+
+const Footer = ({ className }) => {
+    return (
+        <Paper className={className} elevation={0}>
+            <Typography variant="body2" color="textSecondary" align="center">
+                {'Copyright © '}
+                <Link color="inherit" href="https://www.lightmatter.com/">
+                    Lightmatter
+                </Link>{' '}
+                {new Date().getFullYear()}
```

Comment:
Super picky but you could optimize this to have as a good example for people to absorb later. Save this as a var outside of the footer so it doesn't get recalculated every render. SUPER picky :)